<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorTimeScaleForm</class>
 <widget class="QDialog" name="qtractorTimeScaleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>287</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Tempo Map / Markers</string>
  </property>
  <property name="windowIcon">
   <iconset theme="qtractorTracks" />
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="TimeScaleListView">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>120</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Tempo map / Markers</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Bar</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tempo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marker</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="BarTextLabel">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Bar:</string>
         </property>
         <property name="buddy">
          <cstring>BarSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="BarSpinBox">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Bar location</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLabel" name="TimeTextLabel">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>T&amp;ime:</string>
         </property>
         <property name="buddy">
          <cstring>TimeSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="qtractorTimeSpinBox" name="TimeSpinBox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Time/frame location</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="TempoTextLabel">
         <property name="text">
          <string>&amp;Tempo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>TempoSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="qtractorTempoSpinBox" name="TempoSpinBox">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Tempo (BPM) / Time signature</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="2">
        <widget class="QPushButton" name="TempoTapPushButton">
         <property name="text">
          <string>T&amp;ap</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="KeySignatureTextLabel">
         <property name="text">
          <string>&amp;Key signature:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>KeySignatureAccidentalsComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QComboBox" name="KeySignatureAccidentalsComboBox">
         <property name="toolTip">
          <string>Key signature (accidentals)</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="4" colspan="2">
        <widget class="QComboBox" name="KeySignatureModeComboBox">
         <property name="toolTip">
          <string>Key signature (mode)</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minor</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="6">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="6">
        <widget class="QLabel" name="MarkerTextLabel">
         <property name="text">
          <string>&amp;Marker:</string>
         </property>
         <property name="buddy">
          <cstring>MarkerTextLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="5">
        <widget class="QLineEdit" name="MarkerTextLineEdit">
         <property name="toolTip">
          <string>Marker text</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QToolButton" name="MarkerColorToolButton">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Marker color</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="formColor" />
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="6">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLabel" name="TempoFactorTextLabel">
         <property name="text">
          <string>Tempo &amp;scale factor:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>TempoFactorSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QDoubleSpinBox" name="TempoFactorSpinBox">
         <property name="toolTip">
          <string>Tempo scale factor</string>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.01</double>
         </property>
         <property name="maximum">
          <double>10.0</double>
         </property>
         <property name="singleStep">
          <double>0.01</double>
         </property>
         <property name="value">
          <double>1.0</double>
         </property>
        </widget>
       </item>
       <item row="10" column="4" colspan="2">
        <widget class="QPushButton" name="TempoFactorPushButton">
         <property name="text">
          <string>App&amp;ly</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="RefreshPushButton">
       <property name="toolTip">
        <string>Refresh tempo map</string>
       </property>
       <property name="text">
        <string>Re&amp;fresh</string>
       </property>
       <property name="icon">
        <iconset theme="formRefresh" />
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddPushButton">
       <property name="toolTip">
        <string>Add node</string>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset theme="formAdd" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UpdatePushButton">
       <property name="toolTip">
        <string>Update node</string>
       </property>
       <property name="text">
        <string>&amp;Update</string>
       </property>
       <property name="icon">
        <iconset theme="formAccept" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemovePushButton">
       <property name="toolTip">
        <string>Remove node</string>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset theme="formRemove" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClosePushButton">
       <property name="toolTip">
        <string>Close this dialog</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <customwidgets>
  <customwidget>
   <class>qtractorTimeSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtractorSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>qtractorTempoSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtractorSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>TimeScaleListView</tabstop>
  <tabstop>BarSpinBox</tabstop>
  <tabstop>TempoSpinBox</tabstop>
  <tabstop>TempoTapPushButton</tabstop>
  <tabstop>KeySignatureAccidentalsComboBox</tabstop>
  <tabstop>KeySignatureModeComboBox</tabstop>
  <tabstop>MarkerTextLineEdit</tabstop>
  <tabstop>MarkerColorToolButton</tabstop>
  <tabstop>TempoFactorSpinBox</tabstop>
  <tabstop>TempoFactorPushButton</tabstop>
  <tabstop>RefreshPushButton</tabstop>
  <tabstop>AddPushButton</tabstop>
  <tabstop>UpdatePushButton</tabstop>
  <tabstop>RemovePushButton</tabstop>
  <tabstop>ClosePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc"/>
 </resources>
 <connections/>
</ui>
